packages/app/app/ui/page.tsx

```diff
@@ -28,13 +38,13 @@ export default function Page() {
             active={true}
             onClick={() => console.log("hey")}
           >
-            active
+            Active
           </Button>
           <Button size="sm" disabled={true} onClick={() => console.log("hey")}>
-            button disabled
+            Button disabled
           </Button>
         </UISubSection>
-        <UISubSection title="secondary">
+        <UISubSection title="Secondary">
```

Comment:
maybe `UISection` and `UISubSection` should have the style capitalize instead of us making it.

Reply:
It's just a detail. I like it capitalize but we can also have it normal. I think the components should control it, so we can have the freedom to do it as we want :)